src/ape/api/providers.py

```diff
@@ -511,7 +511,7 @@ def prepare_transaction(self, txn: TransactionAPI) -> TransactionAPI:
         """
         return txn
 
-    def get_virtual_machine_error(self, exception: Exception) -> VirtualMachineError:
+    def get_virtual_machine_error(self, exception: Exception, **kwargs) -> VirtualMachineError:
```

Comment:
I might make more sense to move this implementation to `Web3Provider` and keep the one here as either `raises_not_impemented` or abstract